CONTRIBUTING.md

```diff
@@ -69,23 +69,30 @@ To run the test suite on your machine you have multiple options:
 
 - **Full test suite with tox:** Run the full suite across different Python versions with
   
```

Comment:
Personally, I would prefer either not having empty lines in this list before the code blocks, or having empty lines both before and after.

Reply:
No empty lines breaks the conversion to html => empty lines before and after added.